server/index.js

```diff
@@ -10,7 +10,8 @@ const PUBLIC_DIR = path.resolve(__dirname, '..', 'client', 'public');
 
 app.use('/', express.static(PUBLIC_DIR));
 
-app.use('/api/:productId', (req, res) => {
+app.get('/api/products/:productId', (req, res) => {
+  console.log(req.method, 'request at', req.url);
```

Comment:
Keep if needed.

Reply:
Thanks for catching, I'll remove that now.